components/canbus.rst

```diff
@@ -150,20 +150,39 @@ ESP32 CAN Component
 -------------------
 
 The ESP32 has an integrated CAN controller and therefore doesn't need an external controller necessarily.
-You only need to specify the RX and TX pins.
+You only need to specify the RX and TX pins. Any GPIO will work.
 
 .. code-block:: yaml
 
     # Example configuration entry
     canbus:
       - platform: esp32_can
-        tx_pin: GPIO1
-        rx_pin: GPIO3
+        tx_pin: GPIO5
+        rx_pin: GPIO4
         can_id: 4
         bit_rate: 50kbps
         on_frame:
           ...
 
+Wiring options
+**************
+
+5V CAN transceivers are cheap and generate compliant levels. If you power your
+board with 5V this is the preferred option. R501 is important to reduce the 5V
```

Comment:
I'm not super thrilled with the idea of abusing the ESP32's internal ESD diodes like this (with R501), it's using ESP32 out of spec even if it works fine and is likely to be reliable.

If you'd like to change this, let me know, otherwise I'll merge it as-is.

Reply:
Just have a look in the line below there I mention the Voltage divider instead of just one resistor for more advaned Users.

Normally you can find the rating for the internal diodes between 2mA and 40mA in most Microcontroller manuals. Espressive ist known for not providing many data so i die not explicitly checked their datasheet. 
For SPI I use this schematic for years with the ESP.
So for absolute beginners this ist the easiest solution to get a reliable setup without damaging their devices or having to debug rise and fall times behind a Voltage divider.